.github/workflows/deploy.yaml

```diff
@@ -151,23 +171,45 @@ jobs:
           --set secrets.api.GFW_DATA_API_USER_ID=${{ secrets.GFW_DATA_API_USER_ID }}
           --set secrets.api.ELEVENLABS_API_KEY=${{ secrets.ELEVENLABS_API_KEY }}
-      - name: Helm Deploy Zeno Dev Instance
-        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+
+      - name: Helm Deploy Zeno Staging
+        if: github.event_name == 'push' && github.ref == 'refs/heads/develop'
         working-directory: ./helm
         run: >
-          helm upgrade --install zeno-dev ./zeno
-          -f zeno/values-dev.yaml
-          --namespace zeno-dev
-          --create-namespace
-          --set secrets.langfuse.INIT_USER_PASSWORD=${{ secrets.LANGFUSE_INIT_USER_PASSWORD }}
-          --set secrets.langfuse.INIT_PROJECT_SECRET_KEY=${{ secrets.DEV_LANGFUSE_INIT_PROJECT_SECRET_KEY }}
-          --set secrets.langfuse.INIT_PROJECT_PUBLIC_KEY=${{ secrets.DEV_LANGFUSE_INIT_PROJECT_PUBLIC_KEY }}
-          --set secrets.api.OPENAI_API_KEY=${{ secrets.OPENAI_API_KEY }}
-          --set secrets.api.ANTHROPIC_API_KEY=${{ secrets.ANTHROPIC_API_KEY }}
-          --set secrets.api.AWS_ACCESS_KEY_ID=${{ secrets.S3_READONLY_AWS_ACCESS_KEY_ID }}
-          --set secrets.api.AWS_SECRET_ACCESS_KEY=${{ secrets.S3_READONLY_AWS_SECRET_ACCESS_KEY }}
-          --set secrets.api.GEE_SERVICE_ACCOUNT_JSON=${{ secrets.GEE_SERVICE_ACCOUNT_JSON }}
-          --set secrets.api.MAPBOX_API_TOKEN=${{ secrets.MAPBOX_API_TOKEN }}
-          --set secrets.api.GFW_DATA_API_KEY=${{ secrets.GFW_DATA_API_KEY }}
-          --set secrets.api.GFW_DATA_API_USER_ID=${{ secrets.GFW_DATA_API_USER_ID }}
-          --set secrets.api.ELEVENLABS_API_KEY=${{ secrets.ELEVENLABS_API_KEY }}
+          helm upgrade --install zeno ./zeno
+          -f zeno/values.yaml
+          -f zeno/values-${{ env.ENVIRONMENT }}.yaml
+          --set secrets.langfuse.INIT_USER_PASSWORD=${{ secrets.STAGING_LANGFUSE_INIT_USER_PASSWORD }}
+          --set secrets.langfuse.INIT_PROJECT_SECRET_KEY=${{ secrets.STAGING_LANGFUSE_INIT_PROJECT_SECRET_KEY }}
+          --set secrets.langfuse.INIT_PROJECT_PUBLIC_KEY=${{ secrets.STAGING_LANGFUSE_INIT_PROJECT_PUBLIC_KEY }}
+          --set secrets.api.OPENAI_API_KEY=${{ secrets.STAGING_OPENAI_API_KEY }}
+          --set secrets.api.ANTHROPIC_API_KEY=${{ secrets.STAGING_ANTHROPIC_API_KEY }}
+          --set secrets.api.AWS_ACCESS_KEY_ID=${{ secrets.STAGING_S3_READONLY_AWS_ACCESS_KEY_ID }}
+          --set secrets.api.AWS_SECRET_ACCESS_KEY=${{ secrets.STAGING_S3_READONLY_AWS_SECRET_ACCESS_KEY }}
+          --set secrets.api.GEE_SERVICE_ACCOUNT_JSON=${{ secrets.STAGING_GEE_SERVICE_ACCOUNT_JSON }}
+          --set secrets.api.MAPBOX_API_TOKEN=${{ secrets.STAGING_MAPBOX_API_TOKEN }}
+          --set secrets.api.GFW_DATA_API_KEY=${{ secrets.STAGING_GFW_DATA_API_KEY }}
+          --set secrets.api.GFW_DATA_API_USER_ID=${{ secrets.STAGING_GFW_DATA_API_USER_ID }}
+          --set secrets.api.ELEVENLABS_API_KEY=${{ secrets.STAGING_ELEVENLABS_API_KEY }}
```

Comment:
@batpad I don't think these secrets are defined yet. So this helm install step will fail.

Reply:
Yes :) - @leothomas do you feel you can go ahead and populate these in Github Secrets? Happy to sync.